content/release-notes-2020-05-29.md

```diff
@@ -0,0 +1,97 @@
+---
+author: Charlotte Snow and Jim Kreft
+date: 2020-05-29
+slug: release-notes-2020-05-29
+draft: False
```

Comment:
Could you please change this line to `Draft= True`?

Reply:
Yes. It has been changed.

Reply:
I uploaded the new version twice - they are both corrected (I just didn't see it update on the page the first time. Oops.)